cmd/notation/verify.go

```diff
@@ -25,9 +25,18 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 		opts = &verifyOpts{}
 	}
 	command := &cobra.Command{
-		Use:   "verify [flags] <reference>",
-		Short: "Verify Artifacts",
-		Long:  "Verify signatures associated with the artifact.",
+		Use:   "verify [reference]",
+		Short: "Verify OCI artifacts",
+		Long: `Verify OCI artifacts
+
+Prerequisite: added a certificate into trust store and created a trust policy. 
+
+Example - Verify a signature on an OCI artifact identified by a tag  (Notation will resolve tag to digest):
+  notation verify <registry>/<repository>:<tag>
+
+Example - Verify a signature on an OCI artifact identified by a digest:
```

Comment:
suggest move `digest` example to the first one, since it is the recommended way.